examples/web_landing/src/jsMain/kotlin/com/sample/style/Stylesheet.kt

```diff
@@ -7,18 +7,18 @@ object AppCSSVariables : CSSVariables {
     val wtColorGreyLight by variable<Color>()
     val wtColorGreyDark by variable<Color>()
 
-    val wtOffsetTopUnit by variable<CSSSizeValue>()
```

Comment:
This change leads to breaking the web_landing example if it's built using published artifact, doesn't it?
I don't think it's critical. Anyway it will need to get updated very soon after release.

Reply:
I think it's safe enough to swap names for CSSUnit and CSSSize -> in that case anything won't change in that aspect.
Will try after all comments on this PR will be collected )